### PR TITLE
Fix Show/hide filetree button on small displays

### PR DESCRIPTION
**Pull request by sebastian-sauer**
the gt-df's display:flex !important did override the display:none on small displays

image

image

### CODE REVIEW

File: web_src/css/repo.css [hunk 1]
```diff
@@ -2759,6 +2759,7 @@ tbody.commit-list {
   border: none;
   user-select: none;
   color: inherit;
+  display: flex !important;
```
Comment (delvh) [inline web_src/css/repo.css]: Please document why the `!important` is necessary.
Otherwise, we will never know if it is still needed.

Comment (sebastian-sauer) [reply web_src/css/repo.css]: to be honest: i just copied the line from gt-df. Will need to check if the important is really necessary.

Comment (sebastian-sauer) [reply web_src/css/repo.css]: just checked - it's not needed. so i removed it.

**Review by delvh — APPROVED** [review]

**Comment by wxiaoguang** [comment]
It seems that the `gt-ac` and many styles of the class `diff-toggle-file-tree-button` could be simply replaced by `btn interact-fg`

**Comment by sebastian-sauer** [comment]
I've now removed the not needed classes and replaced the css with the helper classes.

**Comment by sebastian-sauer** [comment]
@wxiaoguang is there anything else I should change?

File: templates/repo/diff/box.tmpl [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	<div class="diff-detail-box diff-box">
 		<div class="gt-df gt-ac gt-fw">
 			{{if $showFileTree}}
-				<button class="diff-toggle-file-tree-button gt-df gt-ac not-mobile" data-show-text="{{ctx.Locale.Tr "repo.diff.show_file_tree"}}" data-hide-text="{{ctx.Locale.Tr "repo.diff.hide_file_tree"}}">
+				<button class="diff-toggle-file-tree-button hide-992 btn interact-fg" data-show-text="{{ctx.Locale.Tr "repo.diff.show_file_tree"}}" data-hide-text="{{ctx.Locale.Tr "repo.diff.hide_file_tree"}}">
```
Comment (wxiaoguang) [inline templates/repo/diff/box.tmpl]: Does `not-mobile` fit this case? If the `not-mobile` works well, I think it doesn't need to introduce a new "hide-xxx" system.

Comment (sebastian-sauer) [reply templates/repo/diff/box.tmpl]: not mobile is 200px smaller. but it feels ok to show the file tree even up to the not-mobile breakpoint. 
I'll change this.

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**